elasticdl/python/tests/embedding_test_module.py

```diff
@@ -4,23 +4,65 @@
 from elasticdl.python.elasticdl.layers.embedding import Embedding
 
 
-class CustomModel(tf.keras.Model):
-    def __init__(self, output_dim=16):
-        super(CustomModel, self).__init__(name="embedding_test_model")
+class EdlEmbeddingModel(tf.keras.Model):
+    def __init__(self, output_dim=16, weights=None):
```

Comment:
Add comments stating that weights are the initialization values if set.

Reply:
Added.